tensorflow_text/python/ops/wordpiece_tokenizer_test.py

```diff
@@ -305,6 +305,32 @@ class WordpieceOpTest(ragged_test_util.RaggedTensorTestCase,
           vocab=_DEATH_VOCAB,
           max_bytes_per_word=40,
       ),
+      # Test not splitting out unknown characters.
+      # (p and ! are unknown)
+      dict(
+          tokens=[[b"nap", b"hello!me"]],
+          expected_subwords=[[[b"[UNK]"], [b"[UNK]"]]],
+          unknown_token="[UNK]",
+          vocab=_ENGLISH_VOCAB,
+      ),
+      # Test splitting out unknown characters.
+      dict(
+          tokens=[[b"nap", b"hello!me"]],
+          expected_subwords=[
+              [[b"na", b"[UNK]"], [b"hel", b"##lo", b"[UNK]", b"##me"]]],
```

Comment:
maybe `"##[UNK]"` makes more sense? maybe there should be an argument for `unknown_character` separate to `unknown_token`?

Reply:
+1 to "##[UNK]"; it would make it easier to write a de-tokenizer to stitch back word pieces back into a token.

Reply:
Cool, I changed it to `##[UNK]`

Reply:
can you update this test case? it should be [[b"na", b"##[UNK]"], [b"hel", b"##lo", b"##[UNK]", b"##me"]]] now right?